lib/spack/spack/cmd/style.py

```diff
@@ -61,12 +61,13 @@ def is_package(f):
 
 #: decorator for adding tools to the list
 class tool:
-    def __init__(self, name, required=False):
+    def __init__(self, name, required=False, suffix=".py"):
```

Comment:
This also allows for future tools to be run on files from other languages or for every file (`suffix=""`)

Reply:
Also, I don't think `required` is used in any way?